int32 command_type
bool direction_is_right
int32 speed_r
int32 speed_l

# Constants
float32 TURN_ON_SPOT = 0
float32 STRAIGHT_LINE = -1
float32 STOP = -2
